### DISCUSSION TITLE
Best LLMWare model for RAG

### DISCUSSION THREAD

**Discussion by Preet1234-ux** [llmware-ai/llmware#877] [Q&A]
I have used the model "llmware/bling-phi-3-gguf" for my RAG pipeline . But the accuracy is not good . I have got 66% accuracy for my dataset using this model . I have also seen that if we increase the context retrieval size , the accuracy for some queries was reduced . 
Can you suggest few best models for RAG ?

**Comment by scientific-coder** [comment]
This could be an opportunity to share models that could be interesting to investigate. What about domain specific models for instance ?
I recently discovered [SaulLM](https://huggingface.co/Equall ) with models for legal documents.
What is your dataset about ?

**Comment by Gokul-MK** [comment]
Several models stand out for boosting accuracy in your RAG pipeline. GPT-4-turbo from OpenAI is highly effective for handling large contexts, accommodating up to 32K tokens, making it ideal for nuanced retrieval tasks. Claude 2 by Anthropic, with its capacity for up to 100K tokens, is excellent for managing broader contexts without compromising accuracy. FLAN-T5 by Google is efficient and accurate, particularly when fine-tuned, and provides a good balance for smaller setups.
